site/src/pages/components/switch.explainer.mdx

```diff
@@ -0,0 +1,574 @@
+  of the more intuitive "on"?
+</div>
+
+A `<switch>` can be part of a form and has a mechanism (`type="submit"`) to immediately submit the form it's part of. The value of a switch is reported to the form target as either "on" or "off".
+
+#### Animation
+
+Two popular animation techniques are supported.
+
+##### Background fade
+
+Upon toggling the switch, it's background changes from grey (off) to green (on) and the toggle slides to the respective position. Content on the thumb can indicate the currently active state. Track content is also possible in this variant, but as the track does not move, the content is being hidden and revealed by the sliding thumb.
+
+##### Track slide
+
+Upon toggling the switch, the track and the thumb slide together to the respective position. The track reveals the active side of the track while the inactive track side is being hidden. This variant imitates physical switches and can also reveal track content on either side of the thumb, describing the currently active state.
+
+### Risks and Challenges
+
+{/* _Notable risks or challenges associated with the component. Link issues related to platform blockers._ */}
+
+Implementing the switch as a form element could lead developers to use it as a checkbox replacement.
+
+### Prior Art/Examples
+
+{/* _Link to the OpenUI research page for the component. If any existing, canonical, or exemplary implementations of the component are found beyond what is documented in the research page, make a separate PR to add them there._ */}
+Open UI research page: https://open-ui.org/components/switch/
+
+<figure>
+  <video width="33%" controls>
+    <source src="/images/atlassian_switch_toggle_animation.mp4" type="video/mp4" />
+  </video>
+  <figcaption>
+    Atlassian Design System switch animation with content on the track and a thumb that reveals the
+    active content while hiding the inactive content.
+  </figcaption>
+</figure>
+
+<figure>
+  <video width="33%" controls>
+    <source src="/images/antd-switch-sliding-track-animation.mp4" type="video/mp4" />
+  </video>
+  <figcaption>
+    Various Ant Design switch animation with content on the track and a sliding track animation.
+  </figcaption>
+</figure>
+
+<figure>
+  <video width="33%" controls>
+    <source src="/images/decathlon-switch-thumb-content-animation.mp4" type="video/mp4" />
+  </video>
+  <figcaption>Decathlon Switch example with changing content on the switch thumb.</figcaption>
+</figure>
+
+---
+
+## Design
+
+### User interaction
+
+From a user standpoint, the following interactions should be supported
+
+- Click: toggles the switch
+- Drag & drop: toggles the switch in the direction of the movement, the thumb follows the pointer
+- Swipe: toggles the switch in the direction of the swiping movement, the thumb follows the touch interaction
+- <kbd>Spacebar</kbd>: toggles the switch which has keyboard focus
```

Comment:
I think `Enter` should also work here.

Reply:
I did not include it because it does not work on the checkbox either. Do you want to add it anyways? I initially thought the enter key would toggle a checkbox. A quick google search did not give me the "why" it does not work, there are many people asking how to do that with JavaScript, so it's definitely something that people are building on top of the standard spacebar.

Reply:
if the checkbox were used within a form, Enter would cause it to submit the form.  if a switch can be form associated, it too should mirror checkbox and other form controls that do not use enter to activate/toggle, but instead submit the form.

if a switch is not form associated, then arguably it could work with either space enter or both.  APG notes that switch operates with Spacebar and optionally with Enter as well.